### PR TITLE
Enable color renderable check for fragmentoutput.html

### PR DESCRIPTION
**Pull request by qkmiao**
This aligns with native C++ dEQP. But it doesn't fix any failures.

### CODE REVIEW

File: sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js [hunk 1]
```diff
@@ -177,10 +177,8 @@ var tcuImageCompare = framework.common.tcuImageCompare;
     es3fFragmentOutputTests.FragmentOutputCase.prototype.init = function() {
         // Check that all attachments are supported
         for (var iter = 0; iter < this.m_fboSpec.length; ++iter) {
-            /* TODO: isSizedFormatColorRenderable (in gluTextureUtil) not implemented yet.
-            if (!glu::isSizedFormatColorRenderable(m_context.getRenderContext(), m_context.getContextInfo(), this.m_fboSpec[iter].format))
-                throw tcu::NotSupportedError("Unsupported attachment format");
-                */
+            if (!gluTextureUtil.isSizedFormatColorRenderable(this.m_fboSpec[iter].format))
+                throw new Error('Unsupported attachment format');
```
Comment (qkmiao) [inline sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js]: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/gles3/functional/es3fFragmentOutputTests.cpp#248

File: sdk/tests/deqp/framework/opengl/gluTextureUtil.js [hunk 2]
```diff
@@ -896,138 +896,106 @@ gluTextureUtil.getSampler3D = function(format) {
...
-// }
-
-// /*--------------------------------------------------------------------*//*!
-//  * \brief Check if sized internal format is color-renderable.
-//  * \note Works currently only on ES3 context.
-//  *//*--------------------------------------------------------------------*/
-// bool isSizedFormatColorRenderable (const RenderContext& renderCtx, const ContextInfo& contextInfo, deUint32 sizedFormat)
-// {
-//     deUint32 renderable = 0;
+gluTextureUtil.RenderableType = {
```
Comment (qkmiao) [inline sdk/tests/deqp/framework/opengl/gluTextureUtil.js]: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/framework/opengl/gluTextureUtil.cpp#1064

File: sdk/tests/deqp/framework/opengl/gluTextureUtil.js [hunk 3]
```diff
@@ -896,138 +896,106 @@ gluTextureUtil.getSampler3D = function(format) {
...
+           else
+               return 0;
+
+       // EXT_color_buffer_float / EXT_color_buffer_half_float
+       case gl.RGBA16F:
+       case gl.RG16F:
+       case gl.R16F:
+           if (gl.getExtension("EXT_color_buffer_float") ||
+               gl.getExtension("EXT_color_buffer_half_float"))
+               return gluTextureUtil.RenderableType.RENDERABLE_COLOR;
```
Comment (zhenyao) [inline sdk/tests/deqp/framework/opengl/gluTextureUtil.js]: half_float is not a valid WebGL 2 extension.

**Comment by zhenyao** [comment]
Mostly looks good with one minor issue.

**Comment by qkmiao** [comment]
Thanks for review. I updated this pr. PTAL.

**Comment by zhenyao** [comment]
lgtm thanks

**Pull request merged**